spec.bs

```diff
@@ -1113,7 +1120,7 @@ of the following global objects:
 
     1. Let |result| be [=ParseScript=](|script|, |realm|, `empty`).
 
-       Note: The resulting [=ECMAScript/Script Record=] will have no \[[HostDefined]] component,
+       Note: The resulting [=ECMAScript/Script Record=] will have no [[[=HostDefined=]]] component,

```

Comment:
I don't think the changes here are required?

Reply:
reverted.